CONTRIBUTING.md

```diff
@@ -47,7 +126,7 @@ This will clone the main repo to a local repo on your disk and set up the `origi
 Next you will set up the the second side of the triangle to your fork repo.
 
     cd osgi.enroute.site
-    git remote add fork git@github.com:github-user/osgi.enroute.site.git
+    git remote add fork https://github.com/<YourUserName>/osgi.enroute.site.git
```

Comment:
Since you push to your clone/fork, you should use the ssh URL. I don't think the https URL is to be used for pushing.

Reply:
Well until now I didn't had 2FA ;)
I added a section about setting up SSH and changed the url back to SSH :)